src/common/components/giveFeedbackButton/GiveFeedbackButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import 'hds-core/lib/components/button/button.css';
+
+import styles from './giveFeedbackButton.module.scss';
+
+const feedbackFormLink =
+  // eslint-disable-next-line max-len
+  'https://docs.google.com/forms/d/e/1FAIpQLSdqw2Lq3qooEeRdgr0sV0-Wv-4XcV7IZVzq1HuWoLRa2M7tEg/viewform?usp=pp_url&entry.1982410290=Kulttuurin+kummilapset';
```

Comment:
Even though this feedback implementation is temporary, would it be better if it added in the excel instead? If something happens and it needs to be replaced it would be little easier.

Reply:
My concern with adding this link to the sheet is that once this component is removed, the sheet is not updated and will continue to contain this link. Having the link within this component will make the maintenance chore of removing it easier.

I'm going to confirm that we'll only provide this link in Finnish. If not, I'll manage it through the sheet.

Reply:
When it comes to replacing it, I think that in this scenario the effort is roughly the same. Either the developer finds the correct file and copy pastes the link directly into the source, or they copy paste it into the sheet and pull it with the script. Generally I'd be in favour of storing the "copy" in the same place, so in the sheet, even if it took more effort.

Reply:
We are going to support English speakers as well, so the link will be moved to the sheet :)